fluss-common/src/main/java/org/apache/fluss/metrics/MetricNames.java

```diff
@@ -132,6 +132,8 @@ public class MetricNames {
     public static final String LOG_NUM_SEGMENTS = "numSegments";
     public static final String LOG_END_OFFSET = "endOffset";
     public static final String REMOTE_LOG_SIZE = "size";
+    public static final String LOG_LAKE_PENDING_RECORDS = "pendingRecords";
```

Comment:
Then, the metrics for lake tiering name will be :
- for the timestamp lag(the lag between latest timestamp of lake and fluss): `table_bucket_lakeTiering_timestampLag`
- for the records lag(how many records are not tiered into lake):

@wuchong What do you think of it. Any suggestion on the naming ?